website/source/guides/operations/multi-tenant.html.md

```diff
@@ -846,6 +847,32 @@ field, and **`password`** in the **Password** field.
 
 1. Click **Enable Engine** to finish.
 
+### <a name="step6"></a>Step 6: Audit ambient credentials
+(**Persona:** operator)
+
+Many auth and secrets providers, such as AWS, Azure, GCP, and AliCloud, use ambient
+credentials to authenticate API calls. For example, AWS may:
+
+1. Check for environment variables such as "AWS_SECRET_ACCESS_KEY" and "AWS_ACCESS_KEY_ID".
+1. If not present, use an access key and secret configured in Vault.
+1. If not present, use instance metadata.
```

Comment:
Minor nit -- the order is off here. It's more like:

1. Use an access key and secret key configured in Vault.
1. If not present, check for environment variables such as "AWS_ACCESS_KEY_ID" and "AWS_SECRET_ACCESS_KEY".
1. If not present, load credentials configured in ~/.aws/credentials
1. If not present, use instance metadata.

Reply:
LOL oh yeah, I know, I was giving the AliCloud chain as an example but labelled it AWS for familiarity. However, I see now that it reads strangely so I will give the correct AWS one. Thanks for the feedback!

Reply:
Aha, makes more sense, thanks!